.github/workflows/release-gh-notes.yml

```diff
@@ -0,0 +1,53 @@
+name: release-gh-notes
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        description: "Specify the version for this release"
+        type: string
+      npm_package:
+        required: true
+        description: "npm package of the release"
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: install with retry # Retry 5 times with 1 minute delay to guarantee that the version is released to npm
+        run: |
+          max_retry=5
+          counter=0
+          until npm install ${{ github.event.inputs.npm_package }}@${{ github.event.inputs.version }}
+          do
+            sleep 60
+            [[ counter -eq $max_retry ]] && echo "Failed to install specified version" && exit 1
+            echo "Trying again. Attempt #$counter"
+            ((counter++))
+          done
+        shell: bash
+      - name: Get manifest file
+        id: manifest
+        run: echo ::set-output name=manifest::$(cat node_modules/${{ github.event.inputs.npm_package }}/manifest.json)
+        shell: bash
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ fromJson(steps.manifest.outputs.manifest).commit }}
+          fetch-depth: 0
+      - name: Generate changelog
+        run: npx conventional-changelog-cli -i CHANGELOG.md -s -p conventionalcommits
+      - name: Create Release
+        uses: actions/create-release@v1
```

Comment:
Is this correct link to the repository?

https://github.com/actions/create-release

Looks like it is deprecated

> Please note: This repository is currently unmaintained by a team of developers at GitHub. The repository is here and you can use it as an example, or in Actions. However please be aware that we are not going to be updating issues or pull requests on this repository.

Reply:
replaced with a custom implementation